rules-reviewed/jboss-eap/eap6/eap6.windup.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="eap6"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            This ruleset provides analysis of JBoss EAP 6 xml files that may require
+            individual attention when migrating to JBoss EAP 7+.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
+        </dependencies>
+        <sourceTechnology id="weblogic" />
+        <targetTechnology id="eap" versionRange="[6,)" />
+        <tag>reviewed-2015-06-08</tag>
+    </metadata>
+
+
+    <rules>
+        <rule id="eap6-01000">
+            <when>
+                <javaclass references="{type}">
+                    <location>IMPLEMENTS_TYPE</location>
+                    <location>INHERITANCE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <iteration>
+                    <hint title="CMP Entity EJB" effort="3" severity="mandatory">
+                    <!-- TODO: Migrate this link tom some JBoss + OpenEJB generates it automatically? -->
+                    <message>CMP entity beans are no longer supported in JBoss EAP 7. User is requested to use JPA entity beans that fully replaced the functionality provided by CMP beans.
+		    This entity needs to be migrated using JPA persistence.xml or using JPA annotations.</message>
+                    <link title="CMP Bean to JPA migration" href="http://what-when-how.com/enterprise-javabeans-3/migrating-cmp-2-entity-beans-to-the-ejb-3-jpa-part-1/"/>
```

Comment:
The thing is that I also find a configuration file that configurates these CMP beans. Isn't enough to put about 20 there and then less for each entity? (not sure even if that configuration file is mandatory, just an idea)